bundles/Eiffel/components-configuration.bash

```diff
@@ -94,7 +94,7 @@ server.port=${EIFFEL_EI_FRONTEND_INTERNAL_PORT}
 ei.frontend.service.host=${EIFFEL_EI_FRONTEND}
 ei.frontend.service.port=${EIFFEL_EI_FRONTEND_EXTERNAL_APPLICATION_PORT}
 ei.frontend.context.path=
-ei.use.secure.http.frontend=true
+ei.use.secure.http.frontend=false
```

Comment:
I guess depends how backend is started,, but I guess both works,, worst case is that user might need to accept insecure connection web-browser and/or use --insecure connection flag to curl command when using the EI Application RestAPI and similar rest calls.